packages/framework/src/Component/Elasticsearch/ElasticsearchStructureManager.php

```diff
@@ -85,7 +85,14 @@ protected function getDefinition(int $domainId, string $index): array
     {
         $file = sprintf('%s/%s/%s.json', $this->definitionDirectory, $index, $domainId);
         if (!is_file($file)) {
-            throw new ElasticsearchStructureException(sprintf('File %s not found', $file));
+            throw new ElasticsearchStructureException(
+                sprintf(
+                    'Definition file %d.json, for domain: %d, not found in definition folder "%s". Please make sure that for each domain exists a definition json file with name of a domain ID',
```

Comment:
Personally, I would use the [argument swapping](https://www.php.net/manual/en/function.sprintf.php#refsect1-function.sprintf-examples) for all variables or for none. And I'd prefer to use it because of DRY.

```suggestion
                    'Definition file %1$d.json, for domain: %1$d, not found in definition folder "%2$s". Please make sure that for each domain exists a definition json file with name of a domain ID',
```